modules/codelite/codelite.lua

```diff
@@ -40,7 +40,7 @@
 		local result = value:gsub('&', '&amp;')
 		result = result:gsub('<', '&lt;')
 		result = result:gsub('>', '&gt;')
-		result = result:gsub('"', '\\&quot;')
+		result = result:gsub('"', '&quot;')
```

Comment:
I'm a bit hesitant to approve this, the `\` was required. Does the Premake project still build with CodeLite with this change?

Reply:
For me adding extra `\` break `defines` with quotes and `sharedLib` (on windows).

premake project doesn't use either so it is identical for it.

It is the downside of simple UTs without integration testing ;-)

With my sample project, I now have

`-shared;-Wl,--out-implib="bin/Debug/sharedLib.lib"`
(`"C:/msys64/mingw64/bin/x86_64-w64-mingw32-g++.exe" -shared -fPIC -o bin/Debug/sharedLib.dll @"sharedLib.txt" -LC:\msys64\mingw64\lib  -L.   -shared -Wl,--out-implib="bin/Debug/sharedLib.lib"`
)

instead of previous
`-shared;-Wl,--out-implib=\"bin/Debug/sharedLib.lib\"`
(``"C:/msys64/mingw64/bin/x86_64-w64-mingw32-g++.exe" -shared -fPIC -o bin/Debug/sharedLib.dll @"sharedLib.txt" -LC:\msys64\mingw64\lib  -L.   -shared -Wl,--out-implib=\"bin/Debug/sharedLib.lib\"`
`) with cannot create `"bin/Debug/sharedLib.lib"` (quote in name of file).

Moreover, it makes sense to not have to escape `&quot;` which already mean quote (which might be string delimiter or inside string).

Reply:
> premake project doesn't use either so it is identical for it.

It does, can you answer my question now?

Reply:
I see no problem here.